templates/templates.go

```diff
@@ -55,7 +67,15 @@ func getEnvArray(envName string) ([]string, error) {
 	return strings.Split(env, ","), nil
 }
 
+func required(env interface{}) (string, error) {
+	if env == nil {
+		return "", errors.New("ENV variable is missing")
```

Comment:
Unfortunately, I can't provide information on what ENV is missing as I can't use the second argument - which would be the user's message in case of error" as the JS generated files must not use double quotes.